### ISSUE TITLE
yii\caching\DbCache is incompatible with craft\db\Command

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/cms#5884]
### Description

If you create a `cache` table as specified by the [yii\caching\DbCache::$cacheTable](https://www.yiiframework.com/doc/api/2.0/yii-caching-dbcache#$cacheTable-detail) docs, and configure your `cache` component to use `yii\caching\DbCache`, you will get SQL errors about missing `dateCreated`, `dateUpdated` and `uid` columns.

That happens because `craft\db\Command::upsert()` and `insert()` (which `yii\caching\DbCache` uses) assume that these columns exist, as they do for the vast majority of Craft’s database tables.

These errors can be worked around by adding those three columns to the `cache` table, in addition to the columns Yii requires, however they serve no purpose and actually may slow database caching down, so that’s not a great solution.

Instead we should provide our own `DbCache` class, which extends Yii’s and overrides its `setValue()` and `addValue()` methods, making them compatible with `craft\db\Command` by passing `false` to the `$includeAuditColumns` argument on `upsert()` and `insert()`.

### Additional info

- Craft version: 3.4.13

**Comment by brandonkelly** [comment]
This is now resolved for the next release (3.4.14).

Going forward you will be able to set the `cache` component to a new `craft\cache\DbCache` class, which is fully compatible with `craft\db\Command`.

```php
// -- config/app.php --
return [
    'components' => [
        'cache' => craft\cache\DbCache::class,
    ],
];
```

Also added a new `setup/db-cache-table` command, which can be run to quickly create the `cache` table as specified by [yii\caching\DbCache::$cacheTable](https://www.yiiframework.com/doc/api/2.0/yii-caching-dbcache#$cacheTable-detail).

**Comment by brandonkelly** [comment]
Craft 3.4.14 is out with this change.
